controls/C-0261-satokenmounted.json

```diff
@@ -14,10 +14,10 @@
             }
         ]
     },
-    "description": "",
+    "description": "Check if worload has mounted ServiceAccount token and SA has at least one binding.",
     "remediation": "",
```

Comment:
The description should explain a bit more about the problem and it's impact, you can use the description and remediation from control C-0034, it's pretty similar :)